keras/models.py

```diff
@@ -76,6 +88,49 @@ def standardize_weights(y, sample_weight=None, class_weight=None):
     else:
         return np.ones(y.shape[:-1] + (1,))
 
+def sequential_from_yaml(yamlString):
+    '''
+        Returns a compiled Sequential model generated from a local yaml file,
+        which is either created by hand or from Sequential.to_yaml
+    '''
+    modelYaml = yaml.load(yamlString)
+    model = Sequential()
+
+    class_mode = modelYaml.get('class_mode')
+    theano_mode = modelYaml.get('theano_mode')
+    loss = globals()[modelYaml.get('loss')]
```

Comment:
This would not work with custom loss functions, I believe. It is also somewhat hack-ish and should be avoided.

In fact, why should we compile the model at this stage? Shouldn't we simply return a pre-compilation model, to be compiled by the user with whatever optimizer and loss they see fit?